client/src/partials/permissions/permissions.js

```diff
@@ -11,20 +6,31 @@ PermissionsController.$inject = [
   'ProjectService', 'NodeTreeService'
 ];
 
+/** 
+ * Users and Permission Controller 
+ * 
+ * This module is responsible for handling the creation
+ * of users and assigning permissions to existing modules.
+ *
+ * @todo Password insecure alert or not
+ */
 function PermissionsController($window, $translate, $http, $uibModal, util, Session, Users, Projects, NT) {
   var vm = this;
 
-  var btnTemplate =
-    '<button ng-click="grid.appScope.edit(row.entity)" style="margin: 0 5px">{{ "FORM.EDIT" | translate }}</button>' +
-    '<button ng-click="grid.appScope.editPermissions(row.entity)">{{ "PERMISSIONS.EDIT_PERMISSIONS" | translate }}</button>';
-
+  var editTemplate =
+    '<div style="margin: 0px 5px 5px 5px;"><a class="btn btn-sm btn-default btn-block" ng-click="grid.appScope.edit(row.entity)">{{ "FORM.EDIT" | translate }}</a></div>';
```

Comment:
I think that would be a great idea!

From https://github.com/IMA-WorldHealth/bhima-2.X/pull/217/commits/bf7c3a844ee3972c6d9ef0ef2a2efa9a14572b0a
> Edit and Permissions buttons in the UI grid are now stored in their own
columns and are styled by bootstrap. Notible configuration changes
- Columns are given names with displayName set to ''
- Columns cannot be sorted
- None of the grid columns have menus

> **Inline templates should be extracted to seperate HTML template files in
a further refactor.**